packages/experiments/src/components/PositioningContainer/PositioningContainer.tsx

```diff
@@ -0,0 +1,387 @@
+      }
+    }
+    return this._maxHeight!;
+  }
+
+  private _arePositionsEqual(positions: IPositionInfo, newPosition: IPositionInfo) {
+    return this._comparePositions(positions.calloutPosition, newPosition.calloutPosition);
+  }
+
+  private _comparePositions(oldPositions: ICalloutPositon, newPositions: ICalloutPositon) {
+    for (const key in newPositions) {
+      // This needs to be checked here and below because there is a linting error if for in does not immediately have an if statement
+      if (newPositions.hasOwnProperty(key)) {
+        const oldPositionEdge = oldPositions[key];
+        const newPositionEdge = newPositions[key];
+
+        if (oldPositionEdge && newPositionEdge) {
+          if (oldPositionEdge.toFixed(2) !== newPositionEdge.toFixed(2)) {
+            return false;
+          }
+        }
+      }
+    }
+    return true;
+  }
+
+  private _setTargetWindowAndElement(target: HTMLElement | string | MouseEvent | IPoint | null): void {
+    if (target) {
+      if (typeof target === 'string') {
+        let currentDoc: Document = getDocument()!;
+        this._target = currentDoc ? currentDoc.querySelector(target) as HTMLElement : null;
+        this._targetWindow = getWindow()!;
+      } else if ((target as MouseEvent).stopPropagation) {
+        this._targetWindow = getWindow((target as MouseEvent).toElement as HTMLElement)!;
+        this._target = target;
+      } else if ((target as IPoint).x !== undefined && (target as IPoint).y !== undefined) {
+        this._targetWindow = getWindow()!;
+        this._target = target;
+      } else {
+        let targetElement: HTMLElement = target as HTMLElement;
+        this._targetWindow = getWindow(targetElement)!;
+        this._target = target;
+      }
+    } else {
+      this._targetWindow = getWindow()!;
+    }
+  }
+
+  /**
+   * Animates the height if finalHeight was given.
+   */
+  private _setHeightOffsetEveryFrame(): void {
+    if (this._contentHost && this.props.finalHeight) {
+      this._setHeightOffsetTimer = this._async.requestAnimationFrame(() => {
+        const calloutMainElem = this._contentHost.lastChild as HTMLElement;
+        const cardScrollHeight: number = calloutMainElem.scrollHeight;
+        const cardCurrHeight: number = calloutMainElem.offsetHeight;
+        const scrollDiff: number = cardScrollHeight - cardCurrHeight;
+
+        this.setState({
+          heightOffset: this.state.heightOffset! + scrollDiff
+        });
+
+        if (calloutMainElem.offsetHeight < this.props.finalHeight!) {
+          this._setHeightOffsetEveryFrame();
+        } else {
+          this._async.cancelAnimationFrame(this._setHeightOffsetTimer);
+        }
+      });
+    }
+  }
+
+  private _getTarget(props: IPositioningContainerProps = this.props): HTMLElement | string | MouseEvent | IPoint | null {
+    let { useTargetPoint, targetPoint, target } = props;
+    return useTargetPoint ? targetPoint! : target!;
```

Comment:
Get rid of useTargetPoint, just use target since it can already accept IPoint. useTargetPoint is deprecated.